tests/openthread_ncp/README.md

```diff
@@ -0,0 +1,43 @@
+## OpenThread on RIOT
+
+This test demonstrates how to use OpenThread stack from [OpenThread](https://github.com/openthread/openthread) on
+RIOT. The [Command Line Interface](https://github.com/openthread/openthread/blob/master/examples/apps/cli/README.md) of
+OpenThread was ported. Please check the [full
+documentation](https://github.com/openthread/openthread/blob/master/src/cli/README.md) of the CLI for usage information.
+
+This test show how to use a Network Co-Processor (NCP). A Network Co-Processor is used with wpantund software.
+Wpantund is a user-space network interface driver/daemon that provides a native IPv6 network interface to a low-power
+wireless Network Co-Processor. NCP and wpantund communicates by UART (UART(0) is used here)
+
+## Quick usage
+
+To test OpenThread NCP in RIOT, you can do the following:
+
+1. Flash one node with openthread_ncp example with `make BOARD=<target> clean all flash`
+2. Install wpantund: On Ubuntu 16.04, run
+```
+sudo apt-get -y install build-essential subversion libncurses5-dev libssl-dev zlib1g-dev gawk gcc-multilib flex git-core gettext gcc binutils bzip2 python perl make unzip libz-dev tftp git shtool autogen automake libtool autotools-dev libdbus-1-dev
```

Comment:
I think all this could be added in as a Makefile rule in `dist/tools/wpantund`, see uhcpd/ethos with gnrc_border_router example.
But you can leave it as is for this PR and do that in a follow-up PR.